packages/@aws-cdk/aws-cloudfront/README.md

```diff
@@ -140,6 +140,66 @@ new cloudfront.Distribution(this, 'myDist', {
 });
 ```
 
+### Lambda@Edge
+
+Lambda@Edge is an extension of AWS Lambda, a compute service that lets you execute functions that customize the content that CloudFront delivers.
+You can author Node.js or Python functions in the US East (N. Virginia) region,
+and then execute them in AWS locations globally that are closer to the viewer,
+without provisioning or managing servers.
+Lambda@Edge functions are associated with a specific behavior and event type.
+Lambda@Edge can be used rewrite URLs,
+alter responses based on headers or cookies,
+or authorize requests based on headers or authorization tokens.
+
+The following shows a Lambda@Edge function added to the default behavior and triggered on every request:
+
+```typescript
+const myFunc = new lambda.Function(...);
+new cloudfront.Distribution(this, 'myDist', {
+  defaultBehavior: {
+    origin: cloudfront.Origin.fromBucket(myBucket),
```

Comment:
:)

I _just_ changed the syntax for this. See #9209. The new syntax would use the new cloudfront-origins module.

```suggestion
    origin: new origins.S3Bucket(myBucket),
```

Reply:
I hope you men `new origins.S3Origin(myBucket)`...? I don't see a class `S3Bucket` anywhere.